app/Domain/Mail/MailManager.php

```diff
@@ -147,4 +93,26 @@ private function getUniqueContactsWithPreferredContactType(Integration $integrat
 
         return $uniqueContacts;
     }
+
+    public function sendMail(MailEnabled $event, ?Template $template): ?Integration
+    {
+        if ($template === null || !$template->enabled) {
+            return null;
+        }
+
+        $integration = $this->integrationRepository->getById($event->getId());
+
+        // The technical contact get  additional information in the e-mail (example a link to the satisfaction survey), so this type of contact gets preference when matching email addresses are found
+        foreach ($this->getUniqueContactsWithPreferredContactType($integration, ContactType::Technical) as $contact) {
+            $this->mailer->send(
+                $this->getFrom(),
+                $this->getAddresses($contact),
+                $template->id,
+                $template->subject,
+                $this->getIntegrationVariables($contact, $integration)
+            );
+        }
+
+        return $integration;
```

Comment:
Is this return required?

Reply:
Yes, because for `sendActivationReminderEmail()` I need it to update the ReminderEmailSent.
I wanted to avoid fetching and building the `Integration` twice.

Reply:
Nevermind, going to change it